### ISSUE TITLE
508-defect-2 [HEADINGS]: "Your questionnaire is in progress" should be its own heading or should be placed below the h1

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#24641]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** Josh

## User Story or Problem Statement

As a screen reader user, I use headings to skim and navigate pages.

## Details

Currently, the "Your questionnaire is in progress" blue box is located at the bottom of the page below the `h3` "How will VA protect my personal and health information if I answer these questions?" despite not being a logical child of "How will VA protect my personal and health information..."

https://user-images.githubusercontent.com/14154792/118052257-68af3300-b350-11eb-844c-0f3b3c651ac1.mov

A screen reader may become confused that the continue button is not located beneath the `h1` (which suggests the existence of a CTA nearby it).

To help screen reader users better find and navigate to the questionnaire in progress content, its `strong` should be made into a `h2` OR it should be moved below the `h1`.

## Acceptance Criteria

- [ ] Place the questionnaire in progress content and button below the `h1` OR
- [ ] Convert the questionnaire in progress `strong` into a `h2` (assuming that fixes from other tickets on headings have been made)

### Place the questionnaire in progress content below the `h1`
<img width="827" alt="Screen Shot 2021-05-12 at 6 35 40 PM" src="https://user-images.githubusercontent.com/14154792/118052500-dbb8a980-b350-11eb-8d01-c2a219a22475.png">

### Convert the `strong` into a `h2`

https://user-images.githubusercontent.com/14154792/118052736-3e11aa00-b351-11eb-9c99-afcf015c23a2.mov

Notice now that the `h2` makes the content more discoverable to screen reader users navigating by the headings list.

**Comment by kristenmcconnell** [comment]
@joshkimux In regards to the action link placement on the introduction page, I recently spoke with Danielle about this, since they are taking a holistic look at the intro pages. This whole blue "in-progress" box replaces the main action link when the form is in-progress.

During the initial user test of this feature ([Research Findings](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/questionnaire/research/mvp-usability/Questionnaire%20MVP%20Research%20Findings.pdf)), we had the action at the top and bottom of the page (the traditional layout). Veterans immediately clicked the action at the top of the page and didn't review the rest of the page at all. After we had users slow down and review the content, they did find it valuable information. Therefore, we purposely removed the top call to action and we were able to run another user test. We purposely want them to read the content. The revised intro page as you see it with the action link at the bottom tested very well (with sighted users). I didn't really include this in the [second research report](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/questionnaire/research/all-mvp-workflows/AllMVPWorkflows-Research%20Findings.pdf), because it tested so well.

I know that Riley Orr just ran a test on debts and had a similar finding. She discussed her findings with Danielle and Beth as well.

@mdewey @leahkeeler As a result, I propose that we implement Josh's second suggestion.
 - Convert the questionnaire in progress strong into a h2 (assuming that fixes from other tickets on headings have been made)

This is a platform component within the form's library, correct? Any implications to making this change?

**Comment by mdewey** [comment]
This is a platform change, myself and a few developers are working through how we enhance the platforms to address this. See this slack thread for context https://dsva.slack.com/archives/CBU0KDSB1/p1621531792167200

**Comment by mdewey** [comment]
This is blocked until either 

- [ ] The forms update to the level of headers dynamic updates
- [ ] Platform approves one of the other approaches I have proposed.

**Comment by joshkimux** [comment]
<img width="338" alt="Screen Shot 2021-06-07 at 4 54 20 PM" src="https://user-images.githubusercontent.com/14154792/121086316-01e73300-c7b1-11eb-9ecd-5b43ac8f0351.png">

Validating this has been fixed for screen reader users 👍 

That being said, @kristenmcconnell , from a sighted user's perspective it may be worth deviating from the platform component's usage should the alert be intentionally placed at the bottom based on usability testing. Otherwise, it may appear to sighted users as a child of the previous heading.

<img width="1091" alt="Screen Shot 2021-06-07 at 4 55 19 PM" src="https://user-images.githubusercontent.com/14154792/121086867-b5502780-c7b1-11eb-8c85-e00ca82173f3.png">
<img width="1740" alt="Screen Shot 2021-06-07 at 4 58 18 PM" src="https://user-images.githubusercontent.com/14154792/121086870-b5e8be00-c7b1-11eb-9dab-4907aff05941.png">

**Comment by kristenmcconnell** [comment]
@joshkimux I do really like the visual hierarchy that your suggested change creates. 

@jparker0425 Surfacing this to you to discuss priority of these additional changes compared to check-in work. The questionnaire is only really three questions long, so I'm not sure how many pilot participants will encounter this in progress scenario.

**Comment by jparker0425** [comment]
I agree, I think the suggested changes @joshkimux recommended are necessary and look great, but I do think right now with the timeline we are looking at the check-in work will be the higher priority and we can revisit this in parallel towards the end of the Questionnaire pilot as we get ready to pilot check-in. Thoughts?

**Comment by mdewey** [comment]
That would be another platform-level change. Implementation is a low lift, but I am not sure if we need to run this past any VSP councils or anything.  This would change the header for all forms. I am partial to asking forgiveness and showing a demo of this in action.

**Comment by mdewey** [comment]
I am going to spin this off into a new ticket to focus on the new (and better) recommendations from @joshkimux 

CC: @jparker0425
